fs/_lp_windows.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import { splitList } from "../path/separator.ts";
+import { join } from "../path/win32.ts";
+
+async function chkStat(file: string) {
+  const s = await Deno.stat(file);
+  if (s.isDirectory) {
+    throw new Deno.errors.PermissionDenied(file);
```

Comment:
Maybe better to give more descriptive error message? somthing like `${file} is a directory`